doc/source/tutorials/instrument.rst

```diff
@@ -48,7 +48,7 @@ A good example of a instrument driver is the
 
 Here, let's write a basic example of instrument whose job is to deliver a fixed bias for the duration of the experiment:
 
-.. code-block::  python
+.. testcode::  python
 
     from qibolab.instruments.abstract import Instrument
 
```

Comment:
This `testcode` fails because of `proprietary_instruments`. We should probably revert it back to `code-block`.

Reply:
Indeed, we can skip this one.